src/__experimental__/components/select/__snapshots__/select.spec.js.snap

```diff
@@ -74,26 +74,27 @@ Object {
   "open": true,
   "target": <div
     autocomplete="off"
-    class="sc-htpNat hwJTIW"
+    class="sc-htpNat lhQRHp"
     data-element="input"
+    name="guid-12345"
     role="presentation"
     type="text"
-    value=""
   >
     <input
       autocomplete="off"
       class="sc-bxivhb wHvcT"
       data-element="input"
+      name="guid-12345"
       value=""
     />
     <span
       data-portal-entrance="guid-12345"
     />
     <span
       autocomplete="off"
-      class="sc-ifAKCX jaOJuD"
+      class="sc-ifAKCX hjDnux"
       data-element="input"
-      value=""
+      name="guid-12345"
     >
       <span
         class="carbon-icon icon-dropdown"
```

Comment:
could you import `jest-styled-components` in the `select.spec.js` file to avoid random CSS class names generation?